pebblo/app/service/classification.py

```diff
@@ -0,0 +1,133 @@
+"""
+Module for prompt governance
+"""
+
+import traceback
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, ConfigDict, Field, ValidationError
+
+from pebblo.app.libs.responses import PebbloJsonResponse
+from pebblo.app.models.models import AiDataModel
+from pebblo.entity_classifier.entity_classifier import EntityClassifier
+from pebblo.log import get_logger
+from pebblo.topic_classifier.topic_classifier import TopicClassifier
+
+
+class ClassificationMode(Enum):
+    ENTITY = "entity"
+    TOPIC = "topic"
+    ALL = "all"
+
+
+class ReqClassifier(BaseModel):
+    data: str
+    mode: Optional[ClassificationMode] = Field(default=ClassificationMode.ALL)
```

Comment:
If this is coming in the request then it needs to be string. Consumer of this API may not define `ClassificationMode` class on their own.

Reply:
`ReqClassifier.model_validate()` in L104 correctly maps the enum string (from dict) to enum